frontend/src/lib/utils/icp.utils.ts

```
@@ -48,8 +48,14 @@ export const formatICP = ({
     .replace(/,/g, "'");
 };
 
-export const sumICPs = (...icps: ICP[]): ICP =>
-  ICP.fromE8s(icps.reduce<bigint>((acc, icp) => acc + icp.toE8s(), BigInt(0)));
+export const sumTokenAmounts = (...amountTokens: TokenAmount[]): TokenAmount =>
```

can you prevent the token being mixed?

not sure if we can solve this with typescript generic or if we should throw an error if mixed or if it should return the sum per token

---

Sure, I can make sure all the token symbols are the same.

---

done